docs/whatsnew.rst

```diff
@@ -5,6 +5,8 @@ What's New
 
 These are new features and improvements of note in each release.
 
+.. include:: whatsnew/v0.7.1.txt
```

Comment:
I've seen other packages only include the whatsnew of published versions, so this would be included in the TOC once 0.7.1 is released.